docs/pages/docs/reference/configuration.mdx

```diff
@@ -19,6 +19,34 @@ Defaults to `origin/master`. The base branch or your git repository. Git is used
 A list of globs for implicit global hash dependencies. The contents of these files will be included in the global hashing algorithm.
 This is useful for busting the cache based on `.env` files (not in Git) or any root level file that impacts package tasks (but are not represented in the traditional dependency graph (e.g. a root `tsconfig.json`, `jest.config.js`, `.eslintrc`, etc.)).
 
+## `env`
+
+`type: string`
+
+A list of the names of the environment variables that should be included in the global hashing function (which impacts _all_ tasks).
```

Comment:
Maybe simplify to:

`A list of the names of the environment variables...` -> `A list of environment variables...`